src/ray/protobuf/common.proto

```diff
@@ -212,6 +212,14 @@ message PlacementGroupSpec {
   repeated Bundle bundles = 3;
   // The schedule strategy of this Placement Group.
   PlacementStrategy strategy = 4;
+  // The job id that created this placement group.
+  bytes creator_job_id = 5;
+  // The actor id that created this placement group.
```

Comment:
What will happen if we create a PG in driver?

Reply:
job dead = false
actor dead = true

Reply:
if detached actor;
job dead = true
actor dead = false